Secret-agency-web/src/main/react/secret-agency/tsconfig.json

```diff
@@ -13,7 +13,7 @@
     "forceConsistentCasingInFileNames": true,
     "noImplicitReturns": true,
     "noImplicitThis": true,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```

Comment:
Nedělejme z TypeScriptu Javascript. 🙏

Reply:
nechce sa mi zabavat s modelmi teraz 2 dni pred terminom :D ked sa ti chce mozes to robit tak jak je to koser

Reply:
S kterými? Pro agenta jsem tam nahrával rozpracovaný interface. Tab je jen interface {title: string, link: string}. Žádného komplikované modelu jsem si nevšiml, ale to může být tím, že jsem to je prolítnul. 😄 
Při psaní kódu ti otypovaní objektů ušetří mnohem víc času, než zabere napsání typů. Jelikož ti pak dokáže IDE napovídat. Taky je to bezpečnější.